[2025-01-28T01:37:03.673+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:03.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:37:03.676+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:03.676+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:03.740+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:04.295+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:04.295+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:37:04.330+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:04.330+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 00:00:00+00:00, run_after=2025-01-28 01:00:00+00:00
[2025-01-28T01:37:04.363+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.697 seconds
[2025-01-28T01:37:34.792+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:34.793+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:37:34.796+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:34.795+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:34.820+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:37:34.838+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:34.837+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:37:34.854+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:37:34.854+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:37:34.872+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.084 seconds
[2025-01-28T01:38:05.067+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:05.068+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:38:05.070+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:05.070+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:05.102+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:05.122+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:05.122+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:38:05.143+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:05.142+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:38:05.161+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.098 seconds
[2025-01-28T01:38:35.325+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:35.326+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:38:35.328+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:35.328+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:35.357+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:38:35.375+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:35.374+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:38:35.393+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:38:35.392+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:38:35.411+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.090 seconds
[2025-01-28T01:39:05.585+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:05.586+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:39:05.588+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:05.588+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:05.613+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:05.634+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:05.634+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:39:05.653+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:05.653+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:39:05.669+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.089 seconds
[2025-01-28T01:39:35.932+0000] {processor.py:186} INFO - Started process (PID=93) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:35.933+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:39:35.934+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:35.934+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:35.970+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:39:35.994+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:35.994+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:39:36.028+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:39:36.028+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:39:36.052+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.124 seconds
[2025-01-28T01:46:24.045+0000] {processor.py:186} INFO - Started process (PID=56) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:24.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:46:24.049+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:24.049+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:24.105+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:24.294+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:24.294+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:46:24.330+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:24.330+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:46:24.356+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.315 seconds
[2025-01-28T01:46:54.553+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:54.554+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:46:54.558+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:54.557+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:54.598+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:46:54.625+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:54.625+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:46:54.653+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:46:54.653+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:46:54.675+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.126 seconds
[2025-01-28T01:47:24.799+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:24.799+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:47:24.801+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:24.801+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:24.827+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:24.845+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:24.845+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:47:24.870+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:24.870+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:47:24.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.094 seconds
[2025-01-28T01:47:55.090+0000] {processor.py:186} INFO - Started process (PID=77) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:55.091+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:47:55.093+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:55.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:55.123+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:47:55.137+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:55.137+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T01:47:55.155+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:47:55.155+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 01:00:00+00:00, run_after=2025-01-28 02:00:00+00:00
[2025-01-28T01:47:55.167+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.082 seconds
[2025-01-28T01:49:40.393+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:49:40.394+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:49:40.396+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:49:40.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:49:40.441+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:49:40.430+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:49:40.443+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:49:40.456+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.069 seconds
[2025-01-28T01:50:10.729+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:10.730+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:50:10.733+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:50:10.732+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:10.765+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:50:10.759+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:50:10.767+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:10.782+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.058 seconds
[2025-01-28T01:50:41.064+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:41.064+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:50:41.065+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:50:41.065+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:41.097+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:50:41.090+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:50:41.098+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:50:41.112+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.051 seconds
[2025-01-28T01:51:11.476+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:11.477+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:51:11.478+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:51:11.477+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:11.510+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:51:11.504+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:51:11.512+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:11.526+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.054 seconds
[2025-01-28T01:51:41.805+0000] {processor.py:186} INFO - Started process (PID=79) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:41.806+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:51:41.808+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:51:41.807+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:41.841+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:51:41.835+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:51:41.843+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:51:41.856+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.054 seconds
[2025-01-28T01:52:12.072+0000] {processor.py:186} INFO - Started process (PID=86) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:12.073+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:52:12.074+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:52:12.074+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:12.120+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:52:12.106+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:52:12.121+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:12.134+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.066 seconds
[2025-01-28T01:52:42.417+0000] {processor.py:186} INFO - Started process (PID=93) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:42.418+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:52:42.420+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:52:42.419+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:42.449+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:52:42.442+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:52:42.450+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:52:42.464+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.051 seconds
[2025-01-28T01:53:12.782+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:12.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:53:12.785+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:53:12.785+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:12.815+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:53:12.809+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:53:12.816+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:12.830+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.053 seconds
[2025-01-28T01:53:43.054+0000] {processor.py:186} INFO - Started process (PID=108) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:43.054+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:53:43.055+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:53:43.055+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:43.078+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:53:43.073+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:53:43.079+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:53:43.092+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.043 seconds
[2025-01-28T01:54:13.338+0000] {processor.py:186} INFO - Started process (PID=115) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:13.339+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:54:13.340+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:54:13.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:13.372+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:54:13.367+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:54:13.374+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:13.386+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.054 seconds
[2025-01-28T01:54:43.636+0000] {processor.py:186} INFO - Started process (PID=128) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:43.637+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:54:43.638+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:54:43.638+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:43.671+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:54:43.663+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:54:43.674+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:54:43.689+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.056 seconds
[2025-01-28T01:55:13.912+0000] {processor.py:186} INFO - Started process (PID=135) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:55:13.913+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T01:55:13.914+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:55:13.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:55:13.945+0000] {logging_mixin.py:190} INFO - [2025-01-28T01:55:13.939+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 12, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T01:55:13.947+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T01:55:13.962+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.053 seconds
[2025-01-28T02:46:27.677+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:27.679+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:46:27.681+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:46:27.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:27.727+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:46:27.718+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:46:27.729+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:27.743+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.072 seconds
[2025-01-28T02:46:57.996+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:57.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:46:57.998+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:46:57.998+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:58.026+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:46:58.020+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:46:58.027+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:46:58.041+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.049 seconds
[2025-01-28T02:47:28.324+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:28.325+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:47:28.327+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:47:28.327+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:28.361+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:47:28.354+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:47:28.363+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:28.376+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.058 seconds
[2025-01-28T02:47:58.771+0000] {processor.py:186} INFO - Started process (PID=71) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:58.771+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:47:58.774+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:47:58.774+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:58.806+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:47:58.798+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:47:58.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:47:58.821+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.055 seconds
[2025-01-28T02:48:29.154+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:29.155+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:48:29.157+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:48:29.157+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:29.192+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:48:29.184+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:48:29.193+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:29.209+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.060 seconds
[2025-01-28T02:48:59.584+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:59.585+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:48:59.586+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:48:59.586+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:59.611+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:48:59.606+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:48:59.613+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:48:59.626+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.046 seconds
[2025-01-28T02:49:30.002+0000] {processor.py:186} INFO - Started process (PID=92) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:49:30.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:49:30.005+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:49:30.005+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:49:30.036+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:49:30.029+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:49:30.037+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:49:30.050+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.051 seconds
[2025-01-28T02:50:00.348+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:00.349+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:50:00.352+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:50:00.351+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:00.379+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:50:00.375+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:50:00.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:00.396+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.053 seconds
[2025-01-28T02:50:30.739+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:30.740+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:50:30.742+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:50:30.742+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:30.766+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:50:30.763+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:50:30.768+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:50:30.781+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.045 seconds
[2025-01-28T02:51:01.086+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:01.087+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:51:01.089+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:51:01.088+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:01.120+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:51:01.114+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:51:01.121+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:01.134+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.052 seconds
[2025-01-28T02:51:31.462+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:31.463+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:51:31.465+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:51:31.464+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:31.485+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:51:31.481+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:51:31.486+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:51:31.498+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.039 seconds
[2025-01-28T02:52:01.850+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:01.851+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:52:01.853+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:52:01.852+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:01.881+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:52:01.875+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:52:01.883+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:01.898+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.052 seconds
[2025-01-28T02:52:32.250+0000] {processor.py:186} INFO - Started process (PID=139) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:32.251+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:52:32.252+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:52:32.252+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:32.285+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:52:32.278+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:52:32.287+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:52:32.301+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.055 seconds
[2025-01-28T02:53:02.588+0000] {processor.py:186} INFO - Started process (PID=146) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:02.589+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:53:02.591+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:53:02.590+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:02.622+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:53:02.615+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:53:02.623+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:02.638+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.054 seconds
[2025-01-28T02:53:33.011+0000] {processor.py:186} INFO - Started process (PID=153) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:33.012+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:53:33.015+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:53:33.014+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:33.045+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:53:33.039+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:53:33.047+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:53:33.061+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.054 seconds
[2025-01-28T02:54:03.396+0000] {processor.py:186} INFO - Started process (PID=160) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:03.397+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:54:03.399+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:54:03.399+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:03.429+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:54:03.422+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:54:03.431+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:03.445+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.052 seconds
[2025-01-28T02:54:33.726+0000] {processor.py:186} INFO - Started process (PID=167) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:33.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:54:33.728+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:54:33.728+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:33.758+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:54:33.751+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:54:33.759+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:54:33.774+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.052 seconds
[2025-01-28T02:55:04.026+0000] {processor.py:186} INFO - Started process (PID=174) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:04.026+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:55:04.028+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:55:04.028+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:04.052+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:55:04.046+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:55:04.053+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:04.065+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.044 seconds
[2025-01-28T02:55:34.334+0000] {processor.py:186} INFO - Started process (PID=181) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:34.335+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T02:55:34.337+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:55:34.336+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:34.363+0000] {logging_mixin.py:190} INFO - [2025-01-28T02:55:34.359+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/nasa_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/nasa_dag.py", line 68, in <module>
    insert_data = MongoDBOperator(
                  ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/decorators.py", line 54, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/operators/mongodb_file_operator.py", line 13, in __init__
    with open("/opt/airflow/tmp/data.json", 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/tmp/data.json'
[2025-01-28T02:55:34.364+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T02:55:34.374+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.043 seconds
[2025-01-28T03:05:46.556+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:05:46.557+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:05:46.559+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:05:46.559+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:05:46.598+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:05:46.870+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:05:46.869+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:05:46.895+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:05:46.895+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 02:00:00+00:00, run_after=2025-01-28 03:00:00+00:00
[2025-01-28T03:05:46.915+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.367 seconds
[2025-01-28T03:06:17.377+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:17.378+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:06:17.380+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:17.379+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:17.405+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:17.421+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:17.421+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:06:17.439+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:17.439+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 02:00:00+00:00, run_after=2025-01-28 03:00:00+00:00
[2025-01-28T03:06:17.459+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.086 seconds
[2025-01-28T03:06:47.769+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:47.769+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:06:47.771+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:47.771+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:47.800+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:06:47.815+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:47.815+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:06:47.832+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:06:47.832+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 02:00:00+00:00, run_after=2025-01-28 03:00:00+00:00
[2025-01-28T03:06:47.847+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.082 seconds
[2025-01-28T03:07:18.070+0000] {processor.py:186} INFO - Started process (PID=71) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:07:18.071+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:07:18.073+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:07:18.072+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:07:18.095+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:07:18.114+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:07:18.114+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:07:18.132+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:07:18.132+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:07:18.149+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.084 seconds
[2025-01-28T03:09:17.567+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:17.568+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:09:17.571+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:17.571+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:17.631+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:17.891+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:17.891+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:09:17.911+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:17.910+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:09:17.929+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.370 seconds
[2025-01-28T03:09:48.342+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:48.343+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:09:48.346+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:48.345+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:48.379+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:09:48.404+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:48.404+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:09:48.427+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:09:48.426+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:09:48.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.109 seconds
[2025-01-28T03:10:18.632+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:18.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:10:18.635+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:18.634+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:18.660+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:18.677+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:18.677+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:10:18.696+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:18.696+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:10:18.714+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.085 seconds
[2025-01-28T03:10:48.947+0000] {processor.py:186} INFO - Started process (PID=71) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:48.948+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:10:48.951+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:48.951+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:48.993+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:10:49.020+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:49.020+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:10:49.048+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:10:49.048+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:10:49.069+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.128 seconds
[2025-01-28T03:11:19.300+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:19.301+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:11:19.302+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:19.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:19.331+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:19.352+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:19.352+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:11:19.378+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:19.378+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:11:19.400+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.104 seconds
[2025-01-28T03:11:49.590+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:49.591+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:11:49.593+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:49.592+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:49.618+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:11:49.635+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:49.635+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:11:49.653+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:11:49.653+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:11:49.673+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.087 seconds
[2025-01-28T03:12:19.927+0000] {processor.py:186} INFO - Started process (PID=92) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:19.928+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:12:19.930+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:19.929+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:19.956+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:19.974+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:19.974+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:12:19.992+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:19.992+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:12:20.010+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.086 seconds
[2025-01-28T03:12:50.297+0000] {processor.py:186} INFO - Started process (PID=99) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:50.298+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:12:50.300+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:50.300+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:50.351+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:12:50.380+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:50.380+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:12:50.405+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:12:50.405+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:12:50.426+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.135 seconds
[2025-01-28T03:13:20.599+0000] {processor.py:186} INFO - Started process (PID=112) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:13:20.601+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T03:13:20.604+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:13:20.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:13:20.634+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T03:13:20.655+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:13:20.654+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T03:13:20.678+0000] {logging_mixin.py:190} INFO - [2025-01-28T03:13:20.678+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 03:00:00+00:00, run_after=2025-01-28 04:00:00+00:00
[2025-01-28T03:13:20.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.102 seconds
[2025-01-28T23:56:01.961+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:56:01.963+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T23:56:01.966+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:56:01.966+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:56:02.050+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:56:02.272+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:56:02.272+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T23:56:02.308+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:56:02.307+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 22:00:00+00:00, run_after=2025-01-28 23:00:00+00:00
[2025-01-28T23:56:02.343+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.394 seconds
[2025-01-28T23:58:44.239+0000] {processor.py:186} INFO - Started process (PID=56) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:58:44.241+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T23:58:44.246+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:58:44.245+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:58:44.340+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:58:44.641+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:58:44.641+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T23:58:44.680+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:58:44.680+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 22:00:00+00:00, run_after=2025-01-28 23:00:00+00:00
[2025-01-28T23:58:44.715+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.486 seconds
[2025-01-28T23:59:53.970+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:59:53.971+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/nasa_dag.py for tasks to queue
[2025-01-28T23:59:53.973+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:59:53.973+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:59:54.039+0000] {processor.py:925} INFO - DAG(s) 'nasa_dag' retrieved from /opt/airflow/dags/nasa_dag.py
[2025-01-28T23:59:54.097+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:59:54.097+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-28T23:59:54.135+0000] {logging_mixin.py:190} INFO - [2025-01-28T23:59:54.135+0000] {dag.py:4180} INFO - Setting next_dagrun for nasa_dag to 2025-01-28 22:00:00+00:00, run_after=2025-01-28 23:00:00+00:00
[2025-01-28T23:59:54.171+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/nasa_dag.py took 0.208 seconds
